docs/api/tenacity.md

```diff
@@ -0,0 +1,23 @@
+# `pydantic_ai.tenacity`
+
+## Setup
+
+To use the tenacity integration, you need to install the `tenacity` dependency:
+
+```bash
+pip install tenacity
```

Comment:
```suggestion
pip/uv-add tenacity
```

Reply:
Should this be an optional group on `pydantic-ai` or `pydantic-ai-slim`?

Reply:
Looks like it's already an optional group, so we should use it here

Reply:
should the group be called `tenacity` or `retries`?

Reply:
I prefer `retries`